### ISSUE TITLE
Modularize application

### ISSUE THREAD

**Issue by slaby93** [psmarcin/react-redux-best-practices#5]
As I spoke with Igor, we came up with Idea to split application into modules. In advertisers dashboard we had containers and components, reducers, actions etc folders, mix of all components and logic mostly not related to itself. To avoid wasting time on searching, we can create module folders like :
* Auth
* Creatives
* Applications
* Videos 
etc. Each module will contain related logic, reducers, data models, actions etc.

**Comment by psmarcin** [comment]
Yes, definetely we should split components to functional directories. Thats good idea. Thumb up.
